spark/util.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source /usr/lib/hustler/bin/qubole-bash-lib.sh
+
+##
+# Start Spark History Server
+#
+function start_history_server() {
+   is_master=$(nodeinfo is_master)
+    if [[ "$is_master" == "1" ]]; then
+        monit start sparkhistoryserver
+    fi
+}
+
+##
+# Stop Spark History Server
+#
+function stop_history_server() {
+   is_master=$(nodeinfo is_master)
+    if [[ "$is_master" == "1" ]]; then
+        monit stop sparkhistoryserver
+    fi
```

Comment:
Can return 1 here if it is not master. Otherwise the restart method will unnecessarily wait for 2 seconds.

Reply:
Done